account_invoice_ubl/__manifest__.py

```diff
@@ -15,6 +15,7 @@
         "account_payment_partner",
         "base_ubl_payment",
         "account_tax_unece",
+        "sale",
```

Comment:
Instead of adding a new dependency here, you could wrap your modifications in a block like:

```python
if self.env['ir.module.module'].search([('name', '=', 'sale'), ('state', '=', 'installed')]):
    ...
else:
    ...
```


and in the "else" just keep the old code to avoid change of behavior.